src/models/Lessons.js

```diff
@@ -141,13 +141,13 @@ export function getFirstFutureLesson() {
       },
     },
   )
-    .then(lessons => {console.log(lessons); console.log(lessons[0]); return lessons[0];})
+    .then(lessons => lessons[0])
     .catch(() => 'fail');
```

Comment:
`fail` - isn't too descriptive. You should hide real error from end-user, not from yourself

Reply:
Done.